src/manager.js

```diff
@@ -116,10 +118,19 @@ AdManager.prototype.initAmazonA9 = function() {
  * @returns undefined
 */
 AdManager.prototype.initBaseTargeting = function() {
+  var baseTargeting = targeting.getTargetingPairs().pageOptions;
```

Comment:
This line can probably just go away, and you can change the way `this.targeting` is set during the library initialization.  For instance, this line (https://github.com/theonion/bulbs-public-ads-manager/blob/master/src/manager.js#L24) currently reads:

```
this.targeting = global.TARGETING;
```

but can probably become something like: 
```
this.targeting = global.TARGETING || targeting.getTargetingPairs.pageOptions;
```

That way the implementation of `initBaseTargeting` shouldn't have to change at all.